src/factories/flowRun.ts

```diff
@@ -28,6 +35,23 @@ export async function flowRunContainerFactory() {
     return container
   }
 
+  function toggle(): void {
+    open = !open
+
+    if (open) {
+      drawNodes()
+    }
+
+    container.emit('resized')
+  }
+
+  async function drawNodes(): Promise<void> {
+    const nodes = await nodesContainerFactory('foo')
+    container.addChild(nodes.container)
+
+    nodes.container.on('rendered', () => container.emit('resized'))
+  }
```

Comment:
This will evolve quite a bit. Crud implementation just as a POC of the various pieces being used here.